sklearn/model_selection/tests/test_validation.py

```diff
@@ -1299,3 +1302,19 @@ def test_permutation_test_score_pandas():
         check_series = lambda x: isinstance(x, TargetType)
         clf = CheckingClassifier(check_X=check_df, check_y=check_series)
         permutation_test_score(clf, X_df, y_ser)
+
+
+def test_cross_val_score_labels_paramter():
```

Comment:
checks if setting the classes and priors actually reduce the f1_macro score. because the new class will be given a 0 score in f1_score function.
just wondering, shouldn't the class which is not present be given a 1 and not 0 because it has perfect recall and precision (it wasn't there and wasn't predicted)?